### PR TITLE
Fix Last Commit, removing of new tip

### PR DESCRIPTION
**Pull request by redmangospros**
The last Pr, Removed my tip. I might have done that by mistake.
ping @vutondesign 
/cc @vutondesign

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -41,8 +41,10 @@ <h4> Hosting </h4>
     <ul>
      <li>- Choose whats best for you, hosting your site locally or using services like GitHub Pages or Forge. </li>
      <li>- Make the most of the services your hosting provider gives you. (Like a business email address or image storage.)</li>
-     <li>- Check how much downtime your site host has, if it's a lot then consider moving.</li>
-    </ul>
+    <li>- Check how much downtime your site host has, if it's a lot then consider moving.</li>
+     <li>- Choose a hosting service that is safe and trusted by many people.</li>
```
Comment (vutondesign) [inline index.html]: Indentation is wrong here

File: index.html [hunk 2]
```diff
@@ -41,8 +41,10 @@ <h4> Hosting </h4>
...
+        </ul>
```
Comment (vutondesign) [inline index.html]: ...and here

File: index.html (hunk 1 above)
Comment (redmangospros) [reply index.html]: What do you mean?

Comment (vutondesign) [reply index.html]: There is an extra space before 
```
<li>- Choose a hosting service that is safe and trusted by many people.</li>
```

**Pull request closed without merging**